doozer/doozerlib/cli/scan_sources_konflux.py

```diff
@@ -446,6 +450,52 @@ async def scan_for_config_changes(self, image_meta: ImageMetadata):
                                        RebuildHint(RebuildHintCode.CONFIG_CHANGE,
                                                    'Unable to retrieve config_digest'))
 
+    async def scan_dependency_changes(self, image_meta: ImageMetadata):
+        # Get rebase time from image latest build record
+        build_record = self.latest_image_build_records_map[image_meta.distgit_key]
+        rebase_time = isolate_timestamp_in_release(build_record.release)
+        if not rebase_time:  # no timestamp string in NVR?
+            self.logger.warning('No rebase timestamp string in %s, skipping dependency check', build_record.nvr)
+            return
+        rebase_time = datetime.strptime(rebase_time, "%Y%m%d%H%M%S").replace(tzinfo=timezone.utc)
+
+        # Dependencies are parent images, builders of type member, and operands.
+        dependencies = image_meta.dependencies.copy()
+        base_image = image_meta.config['from'].member
+        if base_image:
+            dependencies.add(base_image)
+        for builder in image_meta.config['from'].builder:
+            if builder.member:
+                dependencies.add(builder.member)
+        self.logger.info('Checking dependencies of %s: %s', image_meta.distgit_key, ','.join(dependencies))
+
+        for dep_key in dependencies:
+            # Is the image dependency included in doozer --images list?
+            if not self.runtime.image_map.get(dep_key, None):
+                self.logger.warning("Image %s has unknown dependency %s. Is it excluded?",
+                                    image_meta.distgit_key, dep_key)
+                return
+
+            # Is the dependency ever been built?
+            dependency_build_record = self.latest_image_build_records_map.get(dep_key, None)
+            if not dependency_build_record:
+                self.logger.warning('Dependency %s of image %s has never been built',
+                                    dep_key, image_meta.distgit_key)
+                return
```

Comment:
same here

Reply:
Fixed in 4dd466c700cc47c439d08b0587ddf397cc9b9538